app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<h2 class="text-center"><%= @recipe.name %> </h2>
+
+<div class="d-flex justify-content-between">
+  <div class="mb-3">
+    <h4 class="pb-3">Preparation Time: <%= @recipe.preparation_time %> <%if @recipe.preparation_time <= 1%> hour <% else %> hours <%end%> </h4>
+    <h4 class="pb-3">Cooking Time: <%= @recipe.cooking_time %> <%if @recipe.cooking_time <= 1%> hour <% else %> hours <%end%> </h4>
+    <p> <%= @recipe.description%> </p>
+  </div>
+  <div>
+    <div class="form-check">
+      <input class="form-check-input" type="checkbox" value="" id="flexCheckCheckedDisabled" <%if @recipe.public %> checked <%end%>  disabled>
+      <label class="form-check" for="flexCheckCheckedDisabled">
```

Comment:
### Your Switch
![image](https://user-images.githubusercontent.com/93039359/219690454-b51f7c38-69fa-4511-a661-36b552516ca8.png)

### Wireframe Switch
![image](https://user-images.githubusercontent.com/93039359/219690359-3b46c00c-8992-400f-8309-1e68a61931f1.png)

- [ ] You can see in the wireframe that you need to have a switch that will make your recipe public and private. It would be nice if you make your checkbox a switch as you can see in the wireframe image above :+1:
- [ ] It would be great if your switch will work as well because now i am not able to change my recipe public status to private :thinking:
- [ ] Kindly make above changes to make your project follow the wireframe and professional :blush:

Reply:
This checkbox has been updated to switch / toggle. However, according to the project requirement, all recipe CRUD are required except update. Therefore, changing the status of a recipe is against the requirement.

<img width="736" alt="Screenshot 2023-02-17 at 10 05 20 PM" src="https://user-images.githubusercontent.com/48407831/219793665-c3cae5e3-61ec-4f5f-9073-5e643421549d.png">

Reply:
resolved! ✔️

Reply:
Hey, @samlorlah I hope you are doing well :blush:
Thanks for mentioning that and accept my apologies for this mistake. It was a requirement before but they have changed it and I didn't notice so it was my mistake and my apologies again.
I would try to be more careful next time.
Best Regards,
Do let me know if you have any question :+1: